WalletWasabi.Fluent/Views/Settings/ThemeChangeView.axaml

```diff
@@ -16,7 +16,6 @@
   <DockPanel VerticalAlignment="Center" Margin="40">
     <TextBlock Text="Wasabi is changing theme..."
                FontSize="20"
```

Comment:
Do we have predefined heading style for this FontSize ?

Reply:
good catch! yeah i'll change this